test/play.py

```diff
@@ -0,0 +1,4 @@
+from animalai.play import play
+
+if __name__ == "__main__":
+    play()
```

Comment:
I'm not sure if this is the unittesting in python. Shoudlnt this be outside of test? And instead have a unittest script ?